riscv-rt/build.rs

```diff
@@ -76,6 +76,13 @@ fn main() {
     let cargo_flags = env::var("CARGO_ENCODED_RUSTFLAGS").unwrap();
     let _name = env::var("CARGO_PKG_NAME").unwrap();
 
+    println!("cargo:rustc-check-cfg=cfg(riscv)");
+    println!("cargo:rustc-check-cfg=cfg(riscv32)");
+    println!("cargo:rustc-check-cfg=cfg(riscv64)");
+    for ext in ['i', 'e', 'm', 'a', 'f', 'd', 'g', 'c'] {
```

Comment:
I like this approach. Could you open a new PR with this modification, please? I think the rest of the crates are already fixed.

Reply:
Oh I just recycled this PR.